### PR TITLE
Using the testHelper from #637 inside of navigation_modify_test

### PR DESCRIPTION
**Pull request by ricknjacky**
issue ref:- #567

### CODE REVIEW

**Comment by ricknjacky** [comment]
image

@moniika as you can clearly see, there are some ESLinting problems I presume.. Any heads up on how to obviate them?
Also, any suggestions on this PR?

File: plugins/keyboard-navigation/test/navigation_modify_test.mocha.js [hunk 1]
```diff
@@ -8,27 +8,10 @@ const chai = require('chai');
...
-    try {
-      console.warn = function(msg) {
-        msgs.push(msg);
-      };
-      innerFunc();
-    } finally {
-      console.warn = nativeConsoleWarn;
-    }
-    return msgs;
-  }
```
Comment (moniika) [inline plugins/keyboard-navigation/test/navigation_modify_test.mocha.js]: The eslint check ("Block must not be padded by blank lines") is complaining about this extra line here.

**Review by moniika — COMMENTED** [review]
> image
> 
> @moniika as you can clearly see, there are some ESLinting problems I presume.. Any heads up on how to obviate them?
> Also, any suggestions on this PR?

Just follow the github action comments (you can see them on the [files changed tab](https://github.com/google/blockly-samples/pull/648/files)). This PR otherwise looks good to me.

**Review by moniika — CHANGES_REQUESTED** [review]
Just fix the eslint issues and it looks good.

**Review by moniika — APPROVED** [review]
Thanks for the PR!

**Pull request merged**